src/managers/message-component-manager.js

```diff
@@ -82,7 +82,7 @@ export function resizeComponent(elementId, size, shouldScale) {
 }
 
 export function loadOverlayComponent(url, message) {
-  document.body.insertAdjacentHTML('beforeend', component(url, message));
+  document.body.insertAdjacentHTML('afterbegin', component(url, message));
```

Comment:
I'm trying to understand this change; what changes if you insert before the node?